app/views/students/status_updates/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Listing Status Updates
 
+= link_to 'New Status Update', [:new, :students, :status_update], class: 'btn btn-default'
+
+p
```

Comment:
I tried a bunch of things, but it won't work with the `br` tag, including throwing errors. We are working on moving the form up here, and loose the button entirely. Okay if we leave it for now? Otherwise we can't ship it today, and I think it is long over due. 
Also, I get 11 warnings in this 25 lines of code file, so I think this file needs more work anyway.